tests/testthat/test-qenv_eval_code.R

```diff
@@ -127,7 +127,7 @@ testthat::test_that("comments alone are pasted to the next/following call elemen
   code <- c("x <- 5", "# comment", "y <- 6")
   q <- eval_code(qenv(), code)
   testthat::expect_identical(
-    unlist(q@code)[2],
+    as.character(q@code)[[2]],
```

Comment:
`as.character` make it already a vector
```suggestion
    as.character(q@code)[2],
```